src/standalone/Soil/energy_hydrology.jl

```diff
@@ -75,6 +75,193 @@ end
 
 Base.broadcastable(ps::EnergyHydrologyParameters) = tuple(ps)
 
+
+### Interfacing with ClimaParams
+
+SoilSurfaceRoughness(::Type{FT}) where {FT <: AbstractFloat} =
+    SoilSurfaceRoughness(CP.create_toml_dict(FT))
```

Comment:
Usually camel case naming is used for types/constructors in Julia ([style guide](https://github.com/CliMA/ClimaCoupler.jl/pull/1318)). Since these functions return NamedTuples rather than objects, it would probably be good to rename them - maybe just to `soil_surface_roughness` and `soil_emissivity`